### ISSUE TITLE
Storage function Authorship.Author empty

### ISSUE THREAD

**Issue by emielvanderhoek** [paritytech/substrate#4021]
Storage function `Authorship`.`Author` returns an empty result. 
According to the documentation I'd expect this to return the current block's author (block producer).
This would be way easier than to find the block_author with the digest logs.

Is this a bug? Should that storage function return the block author?

**Comment by rphmeier** [comment]
No, not a bug. The storage value actually never hits the trie, since we clear it at the end of each block. It's available for runtime processes to inspect, but not committed to the trie as it's technically redundant.

**Comment by rphmeier** [comment]
@emielvanderhoek Did you have a couple extra questions here that we needed to follow up on? I left myself a note to get back to them.

**Comment by emielvanderhoek** [comment]
I figured it out already how to identify the blockAuthor with the Digest Logs. I removed the questions, since they were a bit off-topic for this issue. Thx. (your memory is fine). Your answer closes the issue IMO.
